pennylane/ops/qubit.py

```diff
@@ -1822,6 +1828,61 @@ def adjoint(self):
         return IsingXX(-phi, wires=self.wires)
 
 
+class IsingYY(Operation):
+    r"""IsingYY(phi, wires)
+    Ising YY coupling gate
+
+    .. math:: \mathtt{YY}(\phi) = \begin{bmatrix}
+        \cos(\phi / 2) & 0 & 0 & i \sin(\phi / 2) \\
+        0 & \cos(\phi / 2) & -i \sin(\phi / 2) & 0 \\
+        0 & -i \sin(\phi / 2) & \cos(\phi / 2) & 0 \\
+        i \sin(\phi / 2) & 0 & 0 & \cos(\phi / 2)
+        \end{bmatrix}.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 1
+    * Gradient recipe: :math:`\frac{d}{d\theta}f(YY(\theta)) = \frac{1}{2}\left[f(YY(\theta +\pi/2)) - f(YY(\theta-\pi/2))\right]`
+    where :math:`f` is an expectation value depending on :math:`YY(\phi)`.
```

Comment:
```suggestion
    * Gradient recipe: 
    
    .. math::
         \frac{d}{d\phi} f(YY(\phi)) 
         = \frac{1}{2} \left[ f(YY(\phi + \pi/2)) - f(YY(\phi - \pi/2)) \right]

     where :math:`f` is an expectation value depending on :math:`YY(\phi)`.
```

Reply:
Maybe it would be better to have an indention block for the math equation, just like other `Ising` gates?

I changed `\theta` to `\phi` for consistency.